CrossSolar.Tests/Controller/AnalyticsControllerTests.cs

```diff
@@ -194,5 +196,17 @@ public async Task Post_ShouldInsertOneHourElectricityModel()
             Assert.Equal(201, createdResult.StatusCode);
         }
 
-    }
+        [Fact]
+        public async Task Get_ShouldReturnDayResultsData()
```

Comment:
This test only check if result is OK, but not if correct data are returned. As such provides very little value, since it is proned to both false positives and false negatives. Like here where sollution is incorrect (viz. comment in AnalyticsController), but test still passes.